genesis/src/main.rs

```diff
@@ -97,6 +97,15 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 .required(true)
                 .help("Path to file containing the bootstrap leader's staking keypair"),
         )
+        .arg(
+            Arg::with_name("bootstrap_storage_keypair_file")
+                .short("r")
```

Comment:
actually please just remove the short option.  It's not really helpful

Reply:
Will do. (although I was thinking of it as the "repliaction" account"  :man_shrugging:)